index.html

```diff
@@ -44,6 +44,187 @@ <h3 class="headline-intro">
       
     </div>
     <!--Headline Ends-->
+    <!--Recent Works Section Begins-->
+    <div class="works-container">
+      <div class="works-title">
+        <h2>My Recent Works</h2>
+        <hr>
+      </div>
+      <div class="works-cards">
+        <div class="works-card-1">
+          <img src="./images/Img_Placeholder-card_1.png" alt="card-1-image">
+          <h4>Multi-Post Stories</h4>
+          <div class="card-content-1">A daily selection of privately personalized<br>
+            reads; no accounts or sign-ups<br>
+            has been the industry's standard dummy<br>
+            text ever since the 1500s, when an<br>
+            unknown printer took a standard dummy text.
+          </div>
+          <ul class="tech-stack">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button">See Project</button>
+        </div>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
+          <ul class="tech-stack-2">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="card-2-button" type="button">See Project</button>
+        <div class="works-card-2">
+          <h5>Profesional Art<br>
+            Printing Data
+          </h5>
+          <div class="card-content-2">
+            A daily selection of privately<br>
+            personalized reads; no accounts or<br>
+            sign-ups required. has been the<br>
+            industry's standard
+          </div>
```

Comment:
- [ ] I will suggest you please use a `p` tag here instead of a `div`

Reply:
Hi, i Initially tried to use the p tags  for all of these but none of the content would show, that's why i wrapped it in a div instead.  That's why i didn't use the p tag. Any suggestions on how to fix this?

Reply:
I am sure why that has to occur but I am guessing it's because set the `card-content-2` color to `#fff`which is white color.
Please, try to comment the color line and change the `div` to `p`.
Let me know if that help.